test/extended/util/db_image_helpers.go

```diff
@@ -34,6 +34,12 @@ type MySQL struct {
 	MasterPodName string
 }
 
+// PostgreSQL is a PostgreSQL helper for executing commands
+type PostgreSQL struct {
```

Comment:
do you really need a separate struct for this?  how about renaming the MySQL struct to something more generic, assuming it's even necessary to have a wrapper struct for 2 string values...